app/bundles/EmailBundle/Entity/EmailRepository.php

```diff
@@ -180,11 +180,22 @@ public function getSentReadCount()
      * @param null $listIds
      * @param bool $countOnly
      * @param null $limit
+     * @param int  $minContactId
+     * @param int  $maxContactId
+     * @param bool $countWithMaxMin
      *
      * @return QueryBuilder|int|array
      */
-    public function getEmailPendingQuery($emailId, $variantIds = null, $listIds = null, $countOnly = false, $limit = null)
-    {
+    public function getEmailPendingQuery(
+        $emailId,
+        $variantIds = null,
+        $listIds = null,
+        $countOnly = false,
+        $limit = null,
+        $minContactId = null,
+        $maxContactId = null,
+        $countWithMaxMin = false
```

Comment:
Could you fix code style with CSfixer please?

Reply:
I haven't notice it is styled differently elsewhere. It should be styled in the method body, but not as method params. We can discuss what is better and add the right CS Fixer rule if we find it better.

Reply:
I think you are right. My mistake, sorry.